packages/upload-media/tsconfig.json

```diff
@@ -8,13 +8,13 @@
 	},
 	"include": [ "src/**/*" ],
 	"references": [
-	  { "path": "../api-fetch" },
-	  { "path": "../blob" },
-	  { "path": "../compose" },
-	  { "path": "../data" },
-	  { "path": "../element" },
-	  { "path": "../i18n" },
-	  { "path": "../private-apis" },
-	  { "path": "../url" }
+		{ "path": "../api-fetch" },
+		{ "path": "../blob" },
+		{ "path": "../compose" },
+		{ "path": "../data" },
+		{ "path": "../element" },
+		{ "path": "../i18n" },
+		{ "path": "../private-apis" },
+		{ "path": "../url" }
```

Comment:
It looks like this has been resolved on trunk so we should be able to discard the changes to this file.